src/commands/list.js

```diff
@@ -8,10 +8,17 @@ var Table = require("cli-table");
 var _ = require("lodash");
 var logger = require("../logger");
 
+/**
+ * This command is deprecated in favor of `projects:list` command
+ * TODO: Remove this command 6 months after launching `projects:list` command
+ */
 module.exports = new Command("list")
-  .description("list the Firebase projects you have access to")
+  .description("[DEPRECATED] list the Firebase projects you have access to")
```

Comment:
```suggestion
  .description("[DEPRECATED: use `projects:list`] list the Firebase projects you have access to")
```

A little help for folks reading the description text.

Reply:
Done